src/node/desktop/scripts/fix-library-paths.js

```diff
@@ -0,0 +1,284 @@
+const eachLine = async function (filename, iteratee) {
```

Comment:
I think this script is still a work-in-progress, so I'm not reviewing very closely.

Reply:
That's correct. Everything related to fix library paths is still in progress